elasticsearch/sgconfig/config.yml

```diff
@@ -12,39 +12,12 @@ opendistro_security:
         order: 0
         http_authenticator:
           challenge: false
-          type: io.fabric8.elasticsearch.plugin.auth.OpenShiftTokenAuthentication
+          type: extended-proxy
           config:
-             subjectAccessReviews:
-               prometheus:
-                 verb: get
-                 resource: /metrics
-               jaeger:
-                 verb: get
-                 resource: jaeger
-                 namespace: set-in-run-script
-                 resourceAPIGroup: elasticsearch.jaegertracing.io
-               elasticsearch-operator:
-                 verb: '*'
-                 resource: '*'
-                 namespace: '*'
-                 resourceAPIGroup: logging.openshift.io
+            user_header: 'x-forwarded-user'
+            roles_header: 'x-forwarded-roles'
```

Comment:
please also add in https://github.com/opendistro-for-elasticsearch/security/pull/85/files#diff-062271cb239849b80aa8fb7d9b55487cR217 with a value that will be provided by the 'elasticsearch-proxy' so we are ready for DLS.